packages/cashier/src/components/email-verification-empty-state/__tests__/email-verification-empty-state.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import EmailVerificationEmptyState from '../email-verification-empty-state';
+import { StoreProvider } from '../../../hooks';
```

Comment:
Nit but I suggest adding an alias.

Reply:
@carol-binary I've explained the reason in the comment [above](https://github.com/binary-com/deriv-app/pull/6281#discussion_r995051736) 🙇🏻